roles/build-release/tasks/build.yaml

```diff
@@ -4,18 +4,27 @@
     _deps_file: "ansible-{{ antsibull_ansible_version }}.deps"
     _release_archive: "{{ antsibull_sdist_dir }}/ansible-{{ antsibull_ansible_version }}.tar.gz"
 
+# Documentation for the following commands:
+# https://github.com/ansible-community/antsibull/blob/main/docs/build-ansible.rst
+
 - name: Update version ranges in the build file for alpha and beta releases
   command: >-
     poetry run coverage run -p --source antsibull -m antsibull.cli.antsibull_build new-ansible {{ antsibull_ansible_version }}
       --data-dir {{ antsibull_data_dir }}
   when: antsibull_ansible_version is regex("^\d+.\d+.\d+(a\d+|b1)$")
 
+- name: Set up feature freeze for b2 through rc1
+  set_fact:
+    _feature_freeze: "--feature-frozen"
+  when: antsibull_ansible_version is regex("^\d+.\d+.\d+(b2|b3|rc1)$")
```

Comment:
It occurred to me when I was testing rebuilds yesterday that if there are release candidates after rc1 (say, rc2, rc3) they wouldn't be feature frozen. Should we match on ``^\d+.\d+.\d+(b|rc)\d+$`` ?

Reply:
We already had this discussion two days ago: https://github.com/ansible-community/antsibull/pull/337#discussion_r759441310 :)

Reply:
I'm tired... :zzz: